examples/dt/compute/amd-sev/edpm/openstackdataplanenodeset.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+metadata:
+  name: openstack-edpm-amd-sev
+spec:
+  env:
+    - name: ANSIBLE_FORCE_COLOR
+      value: "True"
+  services:
+    - bootstrap
+    - configure-network
+    - validate-network
+    - install-os
+    - configure-os
+    - run-os
+    - libvirt
+    - ovn
+    - neutron-sriov
+    - neutron-metadata
+    - nova-custom-sriov
```

Comment:
What ever service name you define as part of [examples/dt/compute/amd-sev/edpm/nova_amd.yaml](https://github.com/openstack-k8s-operators/architecture/pull/63/files#diff-32a980a5cf45a9cc69af3f8f3b26e45f5ad5ef2fb8c2a782af5e3da27754f9f4) will need to be used to replace nova-custom-sriov.